oura/oura_server.py

```diff
@@ -26,10 +25,8 @@ def homepage():
 
 def make_authorization_url():
 	# Generate a random string for the state parameter
-	# Save it for use later to prevent xsrf attacks
```

Comment:
I think `xsrf` attacks for a thing which runs for a 30 seconds on localhost is pretty safe.